lib/models/BucketInfo.ts

```diff
@@ -939,4 +949,22 @@ export default class BucketInfo {
         this._capabilities = capabilities;
         return this;
     }
+
+    /**
+     * Get the bucket quota information
+     * @return quotaMax
+     */
+    getQuota() {
+        return this._quotaMax;
+    }
+    
+    /**
+     * Set bucket quota
+     * @param quota - quota to be set
+     * @return - bucket quota info
+     */
+    setQuota(quota: number) {
+        this._quotaMax = quota || 0;
+        return this;
+    }
```

Comment:
Should we add some unit tests here: https://github.com/scality/Arsenal/blob/development/8.1/tests/unit/models/BucketInfo.spec.js

Reply:
https://github.com/scality/Arsenal/pull/2234/commits/0c53d134394fdb2e1815184c80ac1823149cbc02 the tests are already added here and existing ones adapted